README.md

```diff
@@ -1 +1,53 @@
-# frontend
\ No newline at end of file
+# Front end
+
+## To install
+
+### Clone the repo
+```
+git clone https://github.com/Museu-da-Pessoa-XP/frontend
+```
+
+### Install npm
+```
+sudo apt update
+sudo apt install npm
+```
+
+### Install dependecies
+```
+cd frontend
+npm install
+```
+
+## To run app
+```
+npm start
+```
+
+## To build
+```
+npm build
+```
+
+## To test
+```
+npm  test
+```
+
+## Extras 
+
+It is suggested to edit and lint the code with VSCode running with the following extensions. 
```

Comment:
Maybe use a more personal form like "we suggest".

Reply:
Maybe